Fix debian #722621: piuparts says packages use files from /usr/share/doc

Required files are moved to /usr/share/oar

For upstream

Index: oar/Makefiles/api.mk
===================================================================
--- oar.orig/Makefiles/api.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/api.mk	2013-09-19 20:18:52.369312127 +0200
@@ -7,13 +7,14 @@
 
 OARDIR_BINFILES = $(SRCDIR)/oarapi.pl
 
-EXAMPLEDIR_FILES = $(SRCDIR)/oarapi_examples.txt \
-		   $(SRCDIR)/chandler.rb \
-		   $(SRCDIR)/apache2.conf.in \
+SHAREDIR_FILES = $(SRCDIR)/apache2.conf.in \
 		   $(SRCDIR)/api_html_header.pl \
 		   $(SRCDIR)/api_html_postform.pl \
 		   $(SRCDIR)/api_html_postform_resources.pl \
 		   $(SRCDIR)/api_html_postform_rule.pl \
+
+EXAMPLEDIR_FILES = $(SRCDIR)/oarapi_examples.txt \
+		   $(SRCDIR)/chandler.rb \
 		   $(SRCDIR)/INSTALL \
 		   $(SRCDIR)/TODO
 
Index: oar/Makefiles/common.mk
===================================================================
--- oar.orig/Makefiles/common.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/common.mk	2013-09-19 20:18:52.369312127 +0200
@@ -10,7 +10,7 @@
 	       $(SRCDIR)/man/man1/oarprint.1 \
 	       $(SRCDIR)/man/man1/oarnodesetting.1
 
-EXAMPLEDIR_FILES = $(SRCDIR)/tools/oar.conf.in \
+SHAREDIR_FILES = $(SRCDIR)/tools/oar.conf.in \
                    $(SRCDIR)/tools/oarnodesetting_ssh.in \
 		   $(SRCDIR)/tools/update_cpuset_id.sh.in
 
Index: oar/Makefiles/draw-gantt.mk
===================================================================
--- oar.orig/Makefiles/draw-gantt.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/draw-gantt.mk	2013-09-19 20:18:52.373312204 +0200
@@ -1,7 +1,7 @@
 MODULE=draw-gantt
 SRCDIR=sources/visualization_interfaces/DrawGantt
 
-EXAMPLEDIR_FILES=$(SRCDIR)/drawgantt.conf
+SHAREDIR_FILES=$(SRCDIR)/drawgantt.conf
 
 PROCESS_TEMPLATE_FILES= $(SRCDIR)/drawgantt.conf.in \
 			$(SRCDIR)/drawgantt.cgi.in
Index: oar/Makefiles/monika.mk
===================================================================
--- oar.orig/Makefiles/monika.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/monika.mk	2013-09-19 20:18:52.373312204 +0200
@@ -3,7 +3,7 @@
 
 OAR_PERLLIB=$(SRCDIR)/lib
 
-EXAMPLEDIR_FILES= $(SRCDIR)/monika.conf.in \
+SHAREDIR_FILES= $(SRCDIR)/monika.conf.in \
 		  $(SRCDIR)/userInfos.cgi
 
 CGIDIR_FILES = $(SRCDIR)/monika.cgi.in
Index: oar/Makefiles/node.mk
===================================================================
--- oar.orig/Makefiles/node.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/node.mk	2013-09-19 20:18:52.373312204 +0200
@@ -5,7 +5,7 @@
 BINDIR_FILES=$(SRCDIR)/tools/oarnodecheck/oarnodechecklist.in \
 	     $(SRCDIR)/tools/oarnodecheck/oarnodecheckquery.in
 
-EXAMPLEDIR_FILES= $(SRCDIR)/scripts/prologue \
+SHAREDIR_FILES= $(SRCDIR)/scripts/prologue \
 		  $(SRCDIR)/scripts/epilogue \
 		  $(SRCDIR)/tools/sshd_config.in
 
Index: oar/Makefiles/server.mk
===================================================================
--- oar.orig/Makefiles/server.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/server.mk	2013-09-19 20:18:52.373312204 +0200
@@ -35,7 +35,7 @@
 
 SBINDIR_FILES = $(SRCDIR)/server/sbin/oar-server.in
 
-EXAMPLEDIR_FILES = $(SRCDIR)/tools/job_resource_manager.pl \
+SHAREDIR_FILES = $(SRCDIR)/tools/job_resource_manager.pl \
                    $(SRCDIR)/tools/job_resource_manager_cgroups.pl \
 		   $(SRCDIR)/tools/suspend_resume_manager.pl \
 		   $(SRCDIR)/tools/oarmonitor_sensor.pl \
Index: oar/Makefiles/shared/shared.mk
===================================================================
--- oar.orig/Makefiles/shared/shared.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/shared/shared.mk	2013-09-19 20:18:52.373312204 +0200
@@ -75,7 +75,7 @@
 				     $(DOCDIR_FILES) \
 				     $(BINDIR_FILES) \
 				     $(SBINDIR_FILES) \
-				     $(EXAMPLEDIR_FILES) \
+				     $(SHAREDIR_FILES) \
 				     $(CGIDIR_FILES) \
 				     $(WWWDIR_FILES) \
 				     $(MODULE_SETUP_SOURCE_FILES) \
@@ -240,54 +240,54 @@
 	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(SBINDIR)" SOURCE_FILES="$(SBINDIR_FILES)" TARGET_FILE_RIGHTS=0755
 
 #
-# EXAMPLEDIR_FILES
+# SHAREDIR_FILES
 #
 install_examples:
-	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)" SOURCE_FILES="$(EXAMPLEDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)" SOURCE_FILES="$(SHAREDIR_FILES)" TARGET_FILE_RIGHTS=0644
 
 uninstall_examples:
-	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)" SOURCE_FILES="$(EXAMPLEDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)" SOURCE_FILES="$(SHAREDIR_FILES)" TARGET_FILE_RIGHTS=0644
 
 #
 # INITDIR_FILES
 #
 install_init:
-	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/init.d" SOURCE_FILES="$(INITDIR_FILES)" TARGET_FILE_RIGHTS=0755
+	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/init.d" SOURCE_FILES="$(INITDIR_FILES)" TARGET_FILE_RIGHTS=0755
 
 uninstall_init:
-	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/init.d" SOURCE_FILES="$(INITDIR_FILES)" TARGET_FILE_RIGHTS=0755
+	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/init.d" SOURCE_FILES="$(INITDIR_FILES)" TARGET_FILE_RIGHTS=0755
 
 
 #
 # CRONDIR_FILES
 #
 install_cron:
-	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/cron.d" SOURCE_FILES="$(CRONDIR_FILES)" TARGET_FILE_RIGHTS=0644
-	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/cron.hourly" SOURCE_FILES="$(CRONHOURLYDIR_FILES)" TARGET_FILE_RIGHTS=0755
+	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/cron.d" SOURCE_FILES="$(CRONDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/cron.hourly" SOURCE_FILES="$(CRONHOURLYDIR_FILES)" TARGET_FILE_RIGHTS=0755
 
 uninstall_cron:
-	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/cron.d" SOURCE_FILES="$(CRONDIR_FILES)" TARGET_FILE_RIGHTS=0644
-	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/cron.hourly" SOURCE_FILES="$(CRONHOURLYDIR_FILES)" TARGET_FILE_RIGHTS=0755
+	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/cron.d" SOURCE_FILES="$(CRONDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/cron.hourly" SOURCE_FILES="$(CRONHOURLYDIR_FILES)" TARGET_FILE_RIGHTS=0755
 
 
 #
 # DEFAULTDIR_FILES
 #
 install_default:
-	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/default" SOURCE_FILES="$(DEFAULTDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/default" SOURCE_FILES="$(DEFAULTDIR_FILES)" TARGET_FILE_RIGHTS=0644
 
 uninstall_default:
-	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/default" SOURCE_FILES="$(DEFAULTDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/default" SOURCE_FILES="$(DEFAULTDIR_FILES)" TARGET_FILE_RIGHTS=0644
 
 
 #
 # LOGROTATEDIR_FILES
 #
 install_logrotate:
-	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/logrotate.d" SOURCE_FILES="$(LOGROTATEDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_INSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/logrotate.d" SOURCE_FILES="$(LOGROTATEDIR_FILES)" TARGET_FILE_RIGHTS=0644
 
 uninstall_logrotate:
-	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(EXAMPLEDIR)/logrotate.d" SOURCE_FILES="$(LOGROTATEDIR_FILES)" TARGET_FILE_RIGHTS=0644
+	$(SHARED_UNINSTALL) TARGET_DIR="$(DESTDIR)$(SHAREDIR)/logrotate.d" SOURCE_FILES="$(LOGROTATEDIR_FILES)" TARGET_FILE_RIGHTS=0644
 
 #
 # CGIDIR_FILES
Index: oar/Makefiles/www-conf.mk
===================================================================
--- oar.orig/Makefiles/www-conf.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/www-conf.mk	2013-09-19 20:18:52.373312204 +0200
@@ -1,7 +1,7 @@
 MODULE=www-conf
 SRCDIR=sources
 
-EXAMPLEDIR_FILES= $(SRCDIR)/visualization_interfaces/apache.conf.in
+SHAREDIR_FILES= $(SRCDIR)/visualization_interfaces/apache.conf.in
 
 include Makefiles/shared/shared.mk
 
Index: oar/setup/api.sh.in
===================================================================
--- oar.orig/setup/api.sh.in	2013-09-19 11:58:10.389755813 +0200
+++ oar/setup/api.sh.in	2013-09-19 20:18:52.373312204 +0200
@@ -6,27 +6,27 @@
         mv ${OARCONFDIR}/apache-api.conf ${OARCONFDIR}/apache2/oar-restful-api.conf
     fi
     install_conffile \
-        ${EXAMPLEDIR}/apache2.conf \
+        ${SHAREDIR}/apache2.conf \
         ${OARCONFDIR}/apache2/oar-restful-api.conf \
         0600 ${WWWUSER}
 
     install_conffile \
-        ${EXAMPLEDIR}/api_html_header.pl \
+        ${SHAREDIR}/api_html_header.pl \
         ${OARCONFDIR}/api_html_header.pl \
         0600 ${OAROWNER}
 
     install_conffile \
-        ${EXAMPLEDIR}/api_html_postform.pl \
+        ${SHAREDIR}/api_html_postform.pl \
         ${OARCONFDIR}/api_html_postform.pl \
         0644 ${OAROWNER}
 
     install_conffile \
-        ${EXAMPLEDIR}/api_html_postform_resources.pl \
+        ${SHAREDIR}/api_html_postform_resources.pl \
         ${OARCONFDIR}/api_html_postform_resources.pl \
         0644 ${OAROWNER}
 
     install_conffile \
-        ${EXAMPLEDIR}/api_html_postform_rule.pl \
+        ${SHAREDIR}/api_html_postform_rule.pl \
         ${OARCONFDIR}/api_html_postform_rule.pl \
         0644 ${OAROWNER}
 
Index: oar/setup/common.sh.in
===================================================================
--- oar.orig/setup/common.sh.in	2013-09-19 20:18:25.608802112 +0200
+++ oar/setup/common.sh.in	2013-09-19 20:18:52.373312204 +0200
@@ -126,17 +126,17 @@
     mkdir -p ${OARCONFDIR}/
 
     install_conffile \
-        ${EXAMPLEDIR}/oar.conf \
+        ${SHAREDIR}/oar.conf \
         ${OARCONFDIR}/oar.conf \
         0600 ${OAROWNER}:${ROOTGROUP}
 
     install_conffile \
-        ${EXAMPLEDIR}/oarnodesetting_ssh \
+        ${SHAREDIR}/oarnodesetting_ssh \
         ${OARCONFDIR}/oarnodesetting_ssh \
         0755
 
     install_conffile \
-        ${EXAMPLEDIR}/update_cpuset_id.sh \
+        ${SHAREDIR}/update_cpuset_id.sh \
         ${OARCONFDIR}/update_cpuset_id.sh \
         0755
 
Index: oar/setup/draw-gantt.sh.in
===================================================================
--- oar.orig/setup/draw-gantt.sh.in	2013-09-19 11:58:10.389755813 +0200
+++ oar/setup/draw-gantt.sh.in	2013-09-19 20:18:52.373312204 +0200
@@ -2,7 +2,7 @@
 draw_gantt_setup() {
 
     install_conffile \
-        ${EXAMPLEDIR}/drawgantt.conf \
+        ${SHAREDIR}/drawgantt.conf \
         ${OARCONFDIR}/drawgantt.conf \
         0600 \
         ${WWWUSER}
Index: oar/setup/monika.sh.in
===================================================================
--- oar.orig/setup/monika.sh.in	2013-09-19 11:58:10.389755813 +0200
+++ oar/setup/monika.sh.in	2013-09-19 20:18:52.373312204 +0200
@@ -1,7 +1,7 @@
 
 monika_setup() {
     install_conffile \
-        ${EXAMPLEDIR}/monika.conf \
+        ${SHAREDIR}/monika.conf \
         ${OARCONFDIR}/monika.conf \
         0600 \
         ${WWWUSER}
Index: oar/setup/node.sh.in
===================================================================
--- oar.orig/setup/node.sh.in	2013-09-19 11:58:10.389755813 +0200
+++ oar/setup/node.sh.in	2013-09-19 20:18:52.373312204 +0200
@@ -46,15 +46,15 @@
 
 node_setup() {
     install_conffile \
-        ${EXAMPLEDIR}/epilogue \
+        ${SHAREDIR}/epilogue \
         ${OARCONFDIR}/epilogue
     
     install_conffile \
-        ${EXAMPLEDIR}/prologue \
+        ${SHAREDIR}/prologue \
         ${OARCONFDIR}/prologue
 
     install_conffile \
-        ${EXAMPLEDIR}/sshd_config \
+        ${SHAREDIR}/sshd_config \
         ${OARCONFDIR}/sshd_config \
         0600 ${OAROWNER}
 
Index: oar/setup/server.sh.in
===================================================================
--- oar.orig/setup/server.sh.in	2013-09-19 11:58:10.389755813 +0200
+++ oar/setup/server.sh.in	2013-09-19 20:18:52.373312204 +0200
@@ -23,39 +23,39 @@
 
 server_setup() {
     install_conffile \
-        ${EXAMPLEDIR}/job_resource_manager.pl \
+        ${SHAREDIR}/job_resource_manager.pl \
         ${OARCONFDIR}/job_resource_manager.pl \
         0644
 
     install_conffile \
-        ${EXAMPLEDIR}/job_resource_manager_cgroups.pl \
+        ${SHAREDIR}/job_resource_manager_cgroups.pl \
         ${OARCONFDIR}/job_resource_manager_cgroups.pl \
         0644
 
     install_conffile \
-        ${EXAMPLEDIR}/suspend_resume_manager.pl \
+        ${SHAREDIR}/suspend_resume_manager.pl \
         ${OARCONFDIR}/suspend_resume_manager.pl \
         0644
 
     install_conffile \
-        ${EXAMPLEDIR}/oarmonitor_sensor.pl \
+        ${SHAREDIR}/oarmonitor_sensor.pl \
         ${OARCONFDIR}/oarmonitor_sensor.pl \
         0644
 
     install_conffile \
-        ${EXAMPLEDIR}/wake_up_nodes.sh \
+        ${SHAREDIR}/wake_up_nodes.sh \
         ${OARCONFDIR}/wake_up_nodes.sh
 
     install_conffile \
-        ${EXAMPLEDIR}/shut_down_nodes.sh \
+        ${SHAREDIR}/shut_down_nodes.sh \
         ${OARCONFDIR}/shut_down_nodes.sh
 
     install_conffile \
-        ${EXAMPLEDIR}/server_prologue \
+        ${SHAREDIR}/server_prologue \
         ${OARCONFDIR}/server_prologue
 
     install_conffile \
-        ${EXAMPLEDIR}/server_epilogue \
+        ${SHAREDIR}/server_epilogue \
         ${OARCONFDIR}/server_epilogue
 
     setup_oar_ssh
Index: oar/setup/www-conf.sh.in
===================================================================
--- oar.orig/setup/www-conf.sh.in	2013-09-19 11:58:10.389755813 +0200
+++ oar/setup/www-conf.sh.in	2013-09-19 20:18:52.377312280 +0200
@@ -7,7 +7,7 @@
         mv ${OARCONFDIR}/apache.conf ${OARCONFDIR}/apache2/oar-web-status.conf
     fi
     install_conffile \
-        ${EXAMPLEDIR}/apache.conf \
+        ${SHAREDIR}/apache.conf \
         ${OARCONFDIR}/apache2/oar-web-status.conf \
         0644 \
         ${WWWUSER}
Index: oar/Makefile
===================================================================
--- oar.orig/Makefile	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefile	2013-09-19 20:20:43.147423389 +0200
@@ -152,10 +152,12 @@
 	# oar-server
 	mkdir -p $(PACKAGES_DIR)/oar-server/var/lib/oar
 	$(MAKE) -f Makefiles/server.mk $(P_ACTION)\
+    SHAREDIR=/usr/share/oar/oar-server \
                 DESTDIR=$(PACKAGES_DIR)/oar-server
 	
 	$(MAKE) -f Makefiles/database.mk $(P_ACTION)\
                 DESTDIR=$(PACKAGES_DIR)/oar-server \
+    SHAREDIR=/usr/share/oar/oar-server \
 		DOCDIR=/usr/share/doc/oar-server
 	
 	# oar-node
@@ -173,14 +175,17 @@
 	$(MAKE) -f Makefiles/monika.mk $(P_ACTION) \
                 DESTDIR=$(PACKAGES_DIR)/oar-web-status \
 		DOCDIR=/usr/share/doc/oar-web-status \
+		SHAREDIR=/usr/share/oar/oar-web-status \
 		WWWDIR=/usr/share/oar-web-status
 	$(MAKE) -f Makefiles/draw-gantt.mk $(P_ACTION) \
                 DESTDIR=$(PACKAGES_DIR)/oar-web-status \
 		DOCDIR=/usr/share/doc/oar-web-status \
+		SHAREDIR=/usr/share/oar/oar-web-status \
 		WWWDIR=/usr/share/oar-web-status
 	$(MAKE) -f Makefiles/www-conf.mk $(P_ACTION) \
                 DESTDIR=$(PACKAGES_DIR)/oar-web-status \
 		DOCDIR=/usr/share/doc/oar-web-status \
+		SHAREDIR=/usr/share/oar/oar-web-status \
 		WWWDIR=/usr/share/oar-web-status
 	
 	# oar-admin
Index: oar/Makefiles/shared/dist/common.mk
===================================================================
--- oar.orig/Makefiles/shared/dist/common.mk	2013-09-19 11:58:10.389755813 +0200
+++ oar/Makefiles/shared/dist/common.mk	2013-09-19 20:18:52.377312280 +0200
@@ -9,7 +9,7 @@
 export CRONHOURLYDIR?=$(ETCDIR)/cron.hourly
 export LOGROTATEDIR?=$(ETCDIR)/logrotate.d
 export DEFAULTDIR?=$(ETCDIR)/default
-export SHAREDIR?=$(PREFIX)/share/oar-$(strip $(MODULE))
+export SHAREDIR?=$(PREFIX)/share/oar/oar-$(strip $(MODULE))
 export MANDIR?=$(PREFIX)/share/man
 export BINDIR?=$(PREFIX)/bin
 export SBINDIR?=$(PREFIX)/sbin
